src/brevitas/graph/equalize.py

```diff
@@ -1832,7 +1832,10 @@ def find_sink(node):
                 return output_node
 
         for sdpa_node in sdpa_nodes:
-            value_input = sdpa_node.args[-1]
+            if sdpa_node.kwargs.get('value', False):
+                value_input = sdpa_node.kwargs['value']  # value passed as kwarg
+            else:
+                value_input = sdpa_node.args[-1]  # value passed as last arg
```

Comment:
It could be worth checking if we can fully match the signature of the function (we do it for the smoothquant hook), so that we're sure we're picking the correct arg/kwargs.

I am not sure if that works well also with FX nodes though